### PR TITLE
Add full license text

### PR DESCRIPTION
**Pull request by honzajavorek**
The MIT license is not valid and the code is unlicensed (thus cannot be used anywhere and by anyone else) unless it contains a copy of the full license text.

When this gets merged, a new version of the library should be released to npm.

### CODE REVIEW

**Review by kylef — APPROVED** [review]

**Review by ljharb — APPROVED** [review]
I believe the license is still valid enough without the text (because the SPDX field in package.json makes the intent clear) but it’s good to include it just in case of overly paranoid lawyers.

**Comment by kylef** [comment]
@ljharb I believe that isn't legally sufficient. The MIT license itself says that I need to have the copyright notice in any copies. It's a requirement of the license. There would be no copyright notice without the full license and thus render the license invalid:

> Copyright (c) Javier Blanco
> 
> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, *subject to the following conditions*:
>
> **The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.**

The LICENSE file would need to be published within the NPM package so that the copies of it (such as when I `npm install path-parse` I'd have the copyright notice in the LICENSE file).

**Comment by ljharb** [comment]
Yes, i understand that’s the text of the license; I’m saying that when actually tested in court, i believe it would hold up easily with or without including the text.

**Comment by kylef** [comment]
> Yes, i understand that’s the text of the license; I’m saying that when actually tested in court, i believe it would hold up easily with or without including the text.

Perhaps, but that could vary a lot country to country in different legal systems. Unfortunately a lot of lawyers will error on the side of caution blocking use of this library for me at least.

**Comment by kylef** [comment]
I noticed that the maintainer of this library isn't watching it so not getting any notifications. Will just mention @jbgutierrez so he is aware of the PR.

**Pull request merged**
